src/server.js

```diff
@@ -3,34 +3,63 @@
 const express = require('express');
 const { logger } = require('./middleware/logger');
 
+require('./db');
+
 const { hello } = require('./handler/hello');
 const { data } = require('./handler/data');
-
-const makeError = (req, res) => {
-  throw new Error('This is the error handler!');
-};
-
-const pet = (req, res) => {
-  res.status(200).send({name: req.params.pet});
-};
+const { validator }= require('./middleware/validator');
+const {createBoxer, listBoxers, getBoxer, deleteBoxer, updateBoxer } = require('./routes/boxer');
+const { createCoder, listCoders, getCoder, deleteCoder, updateCoder } = require('./routes/coder');
```

Comment:
Replace `const boxerRoute = require('../src/routes/boxer')` and `const boxerRoute = require('../src/routes/coder')`